frontend/src/css/Generic.styl

```diff
@@ -1,21 +1,13 @@
-html
-  height 100%
-
-body
-  min-height 100%
-
-#content
-  height 100%
-
 a
   color fontColor
   &:hover, &:link, &:visited
     color fontColor
     text-decoration none
 
 .fullWidthFullHeight
-  width 100%
-  min-height 100%
+  // width 100%
+  // min-height 100%
```

Comment:
Is this still usefull ?

Reply:
if the new layout works as intended, it's not

Reply:
if the new layout works as intended, it's not

Reply:
removed